pkg/system/installation.go

```diff
@@ -17,35 +19,157 @@ import (
 
 const DBXRootSecret = "yes-i-will-destroy-everything-on-this-disk"
 
-func GetInstallationMode(dbxState dogeboxd.DogeboxState) (dogeboxd.BootstrapInstallationMode, error) {
-	// First, check if we're already installed.
-	if _, err := os.Stat("/opt/dbx-installed"); err == nil {
-		return dogeboxd.BootstrapInstallationModeIsInstalled, nil
-	} else if !os.IsNotExist(err) {
-		return "", fmt.Errorf("error checking for RO installation media: %v", err)
+const (
+	one_gigabyte            = 1024 * 1024 * 1024
+	ten_gigabytes           = 10 * one_gigabyte
+	three_hundred_gigabytes = 300 * one_gigabyte
+	installationMediaPath   = "/opt/ro-media"
```

Comment:
I think this should be clearer, such as 
```suggestion
	isReadOnlyInstallationMediaFile   = "/opt/ro-media"
```

Reply:
Agreed